homeassistant/components/nest.py

```diff
@@ -36,14 +37,23 @@
 
 ATTR_HOME_MODE = 'home_mode'
 ATTR_STRUCTURE = 'structure'
+ATTR_TRIP_ID = 'trip_id'
+ATTR_ETA = 'eta'
+ATTR_ETA_WINDOW = 'eta_window'
+
+HOME_MODE_AWAY = 'away'
+HOME_MODE_HOME = 'home'
 
 SENSOR_SCHEMA = vol.Schema({
     vol.Optional(CONF_MONITORED_CONDITIONS): vol.All(cv.ensure_list)
 })
 
 AWAY_SCHEMA = vol.Schema({
-    vol.Required(ATTR_HOME_MODE): cv.string,
-    vol.Optional(ATTR_STRUCTURE): vol.All(cv.ensure_list, cv.string)
+    vol.Required(ATTR_HOME_MODE): vol.In([HOME_MODE_AWAY, HOME_MODE_HOME]),
+    vol.Optional(ATTR_STRUCTURE): vol.All(cv.ensure_list, cv.string),
+    vol.Optional(ATTR_TRIP_ID): cv.string,
+    vol.Optional(ATTR_ETA): cv.time_period_str,
```

Comment:
Please use a `time_period` instead of `time_period_str`.

Reply:
Fix in #14938